ledger/src/shred.rs

```diff
@@ -115,7 +115,9 @@ impl Shred {
     where
         T: Deserialize<'de>,
     {
-        let ret = bincode::deserialize(&buf[*index..*index + size])?;
+        let ret = bincode::config()
+            .limit(PACKET_DATA_SIZE as u64)
```

Comment:
@pgarg66 can this be limited to shred header size?

Reply:
Probably doesn't need to be more restrictive, we are trying to prevent OOM and packet/blob size is very small in that context